libraries/rush-lib/src/logic/operations/IOperationRunner.ts

```diff
@@ -39,7 +39,38 @@ export interface IOperationRunnerContext {
   /**
    * Object used to track elapsed time.
    */
-  stopwatch: IStopwatchResult;
+  stopwatch: Stopwatch;
```

Comment:
I'd like to elaborate more about the use case. In `CacheableOperationPlugin`, there is a chance that the operation is executing by remote machine. In this case, `stopwatch.reset()` should be invoked to reset the stopwatch. Otherwise, it throws `Call reset() before starting the Stopwatch` in the next time to run the remote executing operation.